app/models/pageflow/entry.rb

```diff
@@ -43,6 +43,7 @@ class PasswordMissingError < StandardError
       create_draft!(home_button_enabled: theming.home_button_enabled_by_default)
       draft.storylines.create!(configuration: {main: true})
       theming.widgets.copy_all_to(draft)
+      theming.copy_defaults_to(draft)
```

Comment:
My preference would be to separate them for now, since one is about defaults and one is about duplicating. But if this grows any bigger it will be ripe for optimization, sure. It doesn't bother me yet.